microsoft/surface-pro/3/default.nix

```diff
@@ -1,9 +1,4 @@
 { lib, ... }:
-
 {
-  # to use the type cover in the initrd
-  boot.kernelModules = [ "hid-microsoft" ];
-
-  # TODO: reverse compat
-  networking.wireless.enable = lib.mkDefault true;
+  imports = [ ../../surface ];
```

Comment:
I think this is better for backwards compatibility.

Reply:
That's actually what I was going to do, heh.

LGTM.